count_ignores.sh

```diff
@@ -0,0 +1,40 @@
+current_dir=$(pwd)
+
+cd exercises
+
+for exercise_name in *
+do
+
+#Get filename
+file_name="$exercise_name/tests/*.rs"
+
+#Commented tests
+comment_test_num=$(cat $file_name | grep "// \#\[test\]" | wc -l)
+
+#Number of tests
+test_num=$(cat $file_name | grep "\#\[test\]" | wc -l)
+test_num=$(($test_num - $comment_test_num))
+
+#Commented ignores
+comment_ignore_num=$(cat $file_name | grep "// \#\[ignore\]" | wc -l)
+
+#Number of ignores
+ignore_num=$(cat $file_name | grep "\#\[ignore\]" | wc -l)
+ignore_num=$(($ignore_num - $comment_ignore_num))
```

Comment:
could consider counting those number of lines for which `#[ignore]` is the first thing on the line (possibly preceded by some number of spaces), rather than taking all lines with `#[ignore]` anywhere on them and subtracting the comments.